mappings/net/minecraft/entity/player/HungerConstants.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_6075 net/minecraft/entity/player/HungerConstants
+	COMMENT Some constants on hunger values.
+	COMMENT
+	COMMENT @see HungerManager
+	FIELD field_30704 FULL_FOOD_LEVEL I
+		COMMENT The maximum food level ({@value}) allowed in a hunger manager.
+	FIELD field_30706 INITIAL_SATURATION_LEVEL F
+		COMMENT The initial saturation level ({@value}) for a newly created hunger manager.
+	FIELD field_30708 EXHAUSION_UNIT F
+		COMMENT A value {@value} that when the exhausion reaches, the exhausion minuses itself
+		COMMENT by to reduce the saturation or food level.
+	FIELD field_30709 SLOW_HEALING_STARVING_INTERVAL I
+		COMMENT When the food tick is a multiple of {@value}, the hunger manager may perform
+		COMMENT slow healing or starving logic.
+	FIELD field_30710 FAST_HEALING_INTERVAL I
+		COMMENT When the food tick is a multiple of {@value}, the hunger manager may perform
+		COMMENT fast healing logic.
+	FIELD field_30711 SLOW_HEALING_FOOD_LEVEL I
+		COMMENT The minimum food level ({@value}) required for the slow-healing mechanism.
```

Comment:
In a previous comment you did not use a hyphen with 'slow healing'.

Reply:
```suggestion
		COMMENT The minimum food level ({@value}) required for the slow healing mechanism.
```